### DISCUSSION TITLE
In the datagrid, text is converted to color brushes, but foreground colors cannot bind fields!

### DISCUSSION THREAD

**Discussion by zy1075984** [AvaloniaUI/Avalonia#12279] [Q&A]
In the datagrid, text is converted to color brushes, but foreground colors cannot bind fields!
`<DataGrid ItemsSource="{Binding lstGateWay}">
        <DataGrid.Columns>
        <DataGridTextColumn Binding="{Binding Status}" Foreground="{Binding Status,Converter={StaticResource StatusToColorConverter}}"/>...`
build error:
	AVLN:0004	Unable to resolve property or method of name 'Status' on type 'FLuX.IoT.DrvAvCrossDef.ViewModels.MainGateWayMagViewModel'.

**Comment by timunie** [comment]
move the binding into a style for DataGridCell.
